lib/msf/core/module/ui/message.rb

```diff
@@ -14,9 +14,8 @@ def print_good(msg='')
 
   def print_prefix
     prefix = ''
-    if (datastore['TimestampOutput'] =~ /^(t|y|1)/i) || (
-      framework && framework.datastore['TimestampOutput'] =~ /^(t|y|1)/i
-    )
+    if datastore['TimestampOutput'] ||
```

Comment:
The globals are special because they're never actually registered anywhere, so no `Option` exists to validate.